update.sh

```diff
@@ -34,24 +34,25 @@ EXPERIMENTAL_APIS=(
     udproutes
 )
 
-rm -rf gateway-api/src/apis/
```

Comment:
What motivated changing the directory structure?

Reply:
I think it was mostly estethics.. I am working on (re)implementing Gateway API in Rust and in my code there was a lot of 
use gateway_api::apis::standard::{}  which seemed a bit superfluous.

Reply:
Yeah, I can understand it seeming superfluous now as the library doesn't have a lot in it except these APIs. The structure was somewhat deliberate however, as there was an anticipation of other facilities (such as the duration mod) that might crop up over time. While I'm not strictly opposed to changing the structure, let's please leave the structure be at least for this PR. Before a v1 release it would definitely be fair to re-assess the layout of things.